client-side/js/tools/meatPatty.js

```diff
@@ -0,0 +1,135 @@
+window.tools.meatPatty = {
+  name: 'meat patty',
+  icon: '/images/meatPatty-icon.png',
+  state: {
+    selected: false,
+    mousePressed: false,
+    mousePosition: { x: null, y: null },
+    colorStop: true
+  },
+  events: {
+    mousedown: function (e) {
+      e.state.mousePressed = true
+      e.state.colorStop = false
+
+      if (e.state.selected) {
+        e.state.mousePosition.x = e.mouse.x
+        e.state.mousePosition.y = e.mouse.y
+        // defines the colors to blend through
+        // red, pink, brown and black
+        const colors = [
+          [255, 71, 71],
+          [232, 192, 173],
+          [102, 41, 0],
+          [0, 0, 0]
+        ]
+        // Declaring the texture image
+        const img = new window.Image()
+        img.src = '/images/meatPatty-texture.png'
```

Comment:
i'm not sure what browser u're using, but i've tested this on Chrome (on Linux) && FireFox (on Linux) && it doesn't work b/c the path to this image doesn't exist. file paths are usually case sensative && ur file name is called "MeathPatty-texture.png" (capital "M") not "meatPatty-texture.png"